packages/bpk-component-graphic-promotion/src/BpkGraphicPromo-test.tsx

```diff
@@ -0,0 +1,123 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import { cssModules } from 'bpk-react-utils';
+
+import BpkGraphicPromo, { TEXT_ALIGN } from './BpkGraphicPromo';
+import type { Props } from './BpkGraphicPromo';
+import STYLES from './BpkGraphicPromo.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+const props: Props = {
+  buttonText: 'Learn more',
+  headline: 'Ride your wave',
+  invertVertically: false,
+  onClick: jest.fn(),
+  sponsor: {
+    label: 'Sponsored',
+    logo: 'path/to/logo.png',
+    altText: 'Airline Name',
+  },
+  subheading:
+    'Portugal and 6 more countries have just been added to the UK travel green list',
+  tagline: 'Tagline',
+  textAlign: TEXT_ALIGN.start,
+};
+
+describe('BpkGraphicPromo', () => {
+  beforeEach(() => {
+    (props.onClick as jest.MockedFunction<Props['onClick']>).mockReset();
+  });
+
+  it('should render correctly with all properties set', () => {
+    const { asFragment } = render(<BpkGraphicPromo {...props} />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
```

Comment:
There had been a test for

```tsx
  it('should support arbitrary props', () => {
    const { asFragment } = render(<BpkGraphicPromo testid="123" {...props} />);

    expect(asFragment()).toMatchSnapshot();
  });
  ```
  
but this proved to be a lie. Looking at the snapshot we can see `testid` was never actually applied. Possibly from an old implementation? The big downside of snapshot tests, people very commonly just press 'update' and don't actually check the diff.

Reply:
Might also be from the default boilerplate tests that get created when you run the new component script